CreateUnitTests.NUnit/NUnit3SolutionManager.cs

```diff
@@ -33,6 +33,9 @@ namespace TestGeneration.Extensions.NUnit
 {
     public class NUnit3SolutionManager : SolutionManagerBase
     {
+        private const string NUnitVersion = "3.12.0";
+        private const string NunitAdapterVersion = "3.13.0";
```

Comment:
This should now be 3.15.1.  

But I thought we had this set to create the latest version ?

Reply:
Fixed.